searx/engines/mojeek.py

```diff
@@ -48,6 +50,13 @@
 language_param = 'lb'
 region_param = 'arc'
 
+_delta_kwargs = {'day': 'days', 'week': 'weeks', 'month': 'months', 'year': 'years'}
```

Comment:
I'm no maintainer, therefore my words may not matter, but my workflow involves looking at the git blame quite regularly. Therefore, I would vouch for this to be a separate commit to the other changes. Otherwise, I agree w/ the improved readability.

Reply:
I squashed the 7 commits into two:

1. squashed patches of the code: 0262d56d0
2. squashed patches generated by `make data.traits`: cc17f80775